python/mage/node_classification/utils/metrics.py

```diff
@@ -0,0 +1,59 @@
+import torch
+import mgp
+from torchmetrics import ConfusionMatrix, Accuracy, AUC, Precision, Recall, F1Score
+from torch_geometric.data import Data
+
+METRICS = {
+    "accuracy": Accuracy,
+    "auc_score": AUC,
+    "precision": Precision,
+    "recall": Recall,
+    "f1_score": F1Score
+}
+
+def metrics(
+    mask: torch.tensor,
+    model: mgp.Any,
+    data: Data,
+    options: mgp.List[str],
+    observed_attribute: str,
+) -> mgp.Map:
+    """Selected metrics calculated for current model and data.
+
+    Args:
+        mask (torch.tensor): used to mask which embeddings should be used
+        model (mgp.Any): model variable
+        data (Data): dataset variable
+        options (mgp.List[str]): list of options to be calculated
+
+    Returns:
+        mgp.Map: dictionary of calculated metrics
+    """
+    model.eval()
+    out = model(data.x_dict, data.edge_index_dict)
```

Comment:
This shouldn't be done here. In metrics you shouldn't push model evaluation. That should be done in top module, and push results here.